### ISSUE TITLE
Multiple output with single input

### ISSUE THREAD

**Issue by techmech46** [cyberbotics/naoqisim#13]
I am using WEBOT simulator. I am trying to run NAO robot. I am using C language. In nao.demo.c, we can move robot forward by clicking forward button, we can move robot left by clicking left button and so on.
Can't we get 2 motion in a single click. Like I want to move robot forward then it will move left. But I want both motion to be done by only one click.
Can we do this?

**Comment by omichel** [comment]
Yes, you can do this. You should edit the [nao_demo.c](https://github.com/cyberbotics/webots/blob/master/projects/robots/softbank/nao/controllers/nao_demo/nao_demo.c) controller and re-program it to execute sequentially several motions instead of one. By the way, you are on the [naoqisim](https://github.com/cyberbotics/naoqisim) repository here, you question should better go to the [webots discussion](https://github.com/cyberbotics/webots/discussions/categories/q-a) instead.
